rosbag2_cpp/include/rosbag2_cpp/compression_options.hpp

```diff
@@ -50,5 +51,14 @@ ROSBAG2_CPP_PUBLIC CompressionMode compression_mode_from_string(const std::strin
  */
 ROSBAG2_CPP_PUBLIC std::string compression_mode_to_string(CompressionMode compression_mode);
 
+/**
+ * Compression options used in the writer which are passed down from the CLI in rosbag2_transport.
+ */
+struct CompressionOptions
+{
+  std::string compression_format;
+  CompressionMode compression_mode;
```

Comment:
I think these can be better named. Is `compression_format` referring to the compression algorithm and `compression_mode` referring to the type of content that's being compressed?

Reply:
Yes, these were the naming conventions back when we updated the bag metadata in #221.